tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "target": "ES2015",
```

Comment:
Some may still need to support IE11, and single-spa generally supports that. I guess we can change this target to es5 if needed in the future though